interpreter/cling/lib/Interpreter/CIFactory.cpp

```diff
@@ -1323,6 +1331,16 @@ namespace {
     if(COpts.CUDAHost)
       argvCompile.push_back("--cuda-host-only");
 
+#ifdef __linux__
```

Comment:
Why is this needed for setting argv but e.g. not for determining `profilingEnabled`? I don't get the plan here, for macOS and Windows: we change the argv flags for `debuggingEnabled` but not for `debuggingEnabled` - why?

Reply:
This part actually doesn't need to be wrapped around the ifdef, as we could enable the frame pointer on macOS, for example, even if the symbol map file is not created (as it's for `perf`, which is Linux-only.

Reply:
The profiling feature is kinda perf-centric right now, which means that macOS and Windows need some work. Maybe we can try with Instruments on macOS and see if the frame-pointer helps and if some other event listener makes sense to help it see JITted code.